login/src/main/resources/templates/web/accounts/new_activation_email.html

```diff
@@ -3,7 +3,27 @@
       xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
       layout:decorator="layouts/main"
       th:with="pivotal=${@environment.getProperty('login.brand') == 'pivotal'},isUaa=${T(org.cloudfoundry.identity.uaa.zone.IdentityZoneHolder).isUaa()}">
+<head>
+    <th:block layout:include="pw_validation :: deps"></th:block>
+    <th:block layout:include="pw_validation :: head"></th:block>
+</head>
 <div class="island-landscape" layout:fragment="page-content">
+    <script th:inline="javascript">
+    /*<![CDATA[*/
+    $(document).ready(function(){
+        // Get all the password policy requirements.
+        // In case the policy equals to null, initialize the values to 0.
+        var special_count = /*[[${passwordPolicy == null ? 0 : passwordPolicy.getRequireSpecialCharacter()}]]*/;
```

Comment:
These should probably be camelCased as it follows the rest of the cloud.gov standard. Unless there's a reason they shouldn't be here?